utilities/EluvioCli.js

```diff
@@ -279,6 +280,10 @@ const CmdTenantFix = async({ argv }) => {
     let errors = res.errors;
     let unresolved = [];
 
+    if (argv.content_admin_address && !t.client.utils.EqualAddress(res.content_admin_address, argv.content_admin_address)) {
```

Comment:
Should we check `res.content_admin_address` is not null address, since `!t.client.utils.EqualAddress(res.content_admin_address, argv.content_admin_address)` will be true and returns error instead of adding the content_admin_address ?

Reply:
Thanks @elv-preethi , you are right. I didn't realize EqualAddress return false when either of the arguments are null. Fixed!